cilium/conntrack.cc

```diff
@@ -190,7 +190,7 @@ uint32_t CtMap::lookupSrcIdentity(const std::string& map_name, const Network::Ad
 
   struct ipv4_ct_tuple key4 {};
```

Comment:
getting `Invalid case style for struct 'ipv4_ct_tuple' (fix available)` lint warning on this type. maybe we should rename these to camel case too (-> `IPv4CtTuple`) ?

Reply:
Applied.